### PR TITLE
Proofreading from Reference > conanfile.py until the end of all docs

### PR DESCRIPTION
**Pull request by dimitrikourk**
First reading done! Please pay particular attention to the FAQ section suggestions. I guess the FAQ is pretty important, so I hope I have not changed the meaning anywhere.

### CODE REVIEW

File: reference/config_files.rst [hunk 1]
```diff
@@ -28,27 +28,27 @@ This is the typical ``~/.conan/conan.conf`` file:
    compiler.version=14
    os=Windows
 
-You can configure here the path where all the packages will be stored (recomended to assign to
-some unit as X: in windows to minimize the issue of long path names)
+Here you can configure the path where all the packages will be stored (it is recomended to assign it to
+some unit, e.g. map it to X: on windows, in order to avoid having to work with long path names).
```
Comment (memsharded) [inline reference/config_files.rst]: Not that we dont want to work with long paths, but the issue of max 250 chars in win path names.

**Comment by memsharded** [comment]
Excellent!! Great work, thanks very much.

File: reference/config_files.rst (hunk 1 above)
Comment (dimitrikourk) [inline reference/config_files.rst]: I see. And on Windows they tend to get pretty long indeed :)

**Comment by dimitrikourk** [comment]
It's a pleasure! Now I will actually start using the tool :D

**Pull request merged**